less/_components/classes-flex.less

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2018 Eclipse Foundation, Inc.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * Contributors:
+ *   Christopher Guindon <chris.guindon@eclipse-foundation.org>
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+*/
+
+// Classes styles: Flex
+// css classes for Flex layout.
+
+.display-flex {
+  display: flex;
+}
+
+.vertical-align {
```

Comment:
this should probably be left in grids less. While this has flex, it isn't a flex feature and is more using it as a part of the solution.

Reply:
moved back